bom/pom.xml

```diff
@@ -163,6 +163,8 @@
         <keycloak.version>6.0.1</keycloak.version>
         <subethasmtp.version>3.1.7</subethasmtp.version>
         <hibernate-quarkus-local-cache.version>0.1.0</hibernate-quarkus-local-cache.version>
+        <kogito.version>0.1.1</kogito.version>
+        <swagger.version>1.5.21</swagger.version>
```

Comment:
I would prefer to avoid having the Swagger annotations and use OpenAPI instead. That's what we want to use and everything will be handled by our OpenAPI extension and our OpenAPI/Swagger UI integration.

Reply:
this is on our roadmap for next release, can we stick to swagger for now and replace it with next update?

Reply:
Fair enough!

Reply:
thanks, anything else missing from your point of view?